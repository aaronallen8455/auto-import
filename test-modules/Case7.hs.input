module Case7 where

-- comment
import           Data.Void

-- | doc
main :: IO ()
main = TIO.putStrLn "hi"
