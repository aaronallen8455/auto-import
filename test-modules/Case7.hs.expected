module Case7 where

-- comment
import qualified Data.Text.IO as TIO
import           Data.Void

-- | doc
main :: IO ()
main = TIO.putStrLn "hi"
